src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/PlacementRequestTransformer.kt

```diff
@@ -24,9 +25,13 @@ class PlacementRequestTransformer(
   private val assessmentTransformer: AssessmentTransformer,
   private val userTransformer: UserTransformer,
   bookingSummaryTransformer: BookingSummaryTransformer,
+  cas1SpaceBookingSummaryTransformer: Cas1SpaceBookingSummaryTransformer,
 ) {
 
-  val placementRequestBookingSummaryTransformer = PlacementRequestBookingSummaryTransformer(bookingSummaryTransformer)
+  val placementRequestBookingSummaryTransformer = PlacementRequestBookingSummaryTransformer(
+    bookingSummaryTransformer,
+    cas1SpaceBookingSummaryTransformer,
+  )
```

Comment:
sorry about this awkwardness of instantiating a transformer inline, it was a quick fix by me to avoid having to rewrite a bunch of the unit tests for this class (which are very mock heavy!). Given we're probably going to rewrite CRU dashboard I think this is acceptable